fe/fe-core/src/main/java/org/apache/doris/persist/MetaCleaner.java

```diff
@@ -67,6 +67,17 @@ public void clean() throws IOException {
             }
         }
     }
+
+    public void cleanTheLatestInvalidImageFile(String path) throws IOException {
+        File latestInvalidImage = new File(path);
+        if (latestInvalidImage.exists()) {
+            if (latestInvalidImage.delete()) {
+                LOG.info(latestInvalidImage.getAbsoluteFile() + " deleted.");
+            } else {
+                LOG.warn(latestInvalidImage.getAbsoluteFile() + " delete failed.");
```

Comment:
Can it print the reason to the warning log if delete failed

Reply:
https://github.com/apache/incubator-doris/blob/87fc46f84c81c197607bccfa68056209ba226f6c/fe/fe-core/src/main/java/org/apache/doris/master/Checkpoint.java#L161-L166

It will print the exception to the warning log as error.